Source/WebCore/platform/mediastream/mac/ScreenCaptureKitCaptureSource.h

```diff
@@ -75,6 +75,14 @@ class ScreenCaptureKitCaptureSource final
     static std::optional<CaptureDevice> screenCaptureDeviceWithPersistentID(const String&);
     static std::optional<CaptureDevice> windowCaptureDeviceWithPersistentID(const String&);
 
+    enum SnapshotOptions {
```

Comment:
shouldn't this be an enum class ?
They are used with fully qualified name later.